modules/odc-customization-of-user-perspectives.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+[id="odc-customization-of-user-perspectives_{context}"]
+= Customization of user perspectives 
```

Comment:
The mod docs guide does say

"In such cases, ensure that the heading of the concept or reference remains a noun phrase and not a gerund. "

But in this case, maybe we should make an exception, especially seeing as we are using the CONCEPT as a workaround for not being having nested procedure modules. I think the title would look better as "Customizing user perspectives" in the toc, rather than as a noun phrase.

```
Adding a custom logo and product name
Creating custom links in the web console
Customizing console routes
  Customizing the console route
  Customizing the download route
Customizing the login page
Defining a template for an external log link
Creating custom notification banners
Customizing CLI downloads
Adding YAML examples to Kubernetes resources
Customization of user perspectives
  Customizing a perspective using YAML view
  Customizing a perspective using form view
```

Reply:
@gabriel-rh, Incorporated this suggestion. Looks completely justified.